core/src/main/java/org/bitcoinj/utils/Network.java

```diff
@@ -23,6 +23,7 @@
  */
 public enum Network {
     MAIN(NetworkParameters.ID_MAINNET),
+    @Deprecated
```

Comment:
This small change seems unrelated. I'm not opposed to it, shall we merge this right away?

Reply:
Or maybe it is related, I'm not sure.

Reply:
It's not really related, but if we're gong to use `Network` enum more seriously, it would be a good idea if it doesn't have two different values that are "the same".

Reply:
See PR #2434

Reply:
Removed in favor of PR #2434.